views/email/a/1_common.tx

```diff
@@ -3,17 +3,17 @@
     <p style="margin-bottom: 1em">That's because Incognito mode <strong>isn't</strong> really private. <img class="emoticon" src="https://duckduckgo.com/assets/email/shocked1.png?atb=<: $subscriber.flow :>" height="20" width="20" alt="shocked face emoji" />
 </p>
 <p style="margin-bottom: 1em">
-    Incognito mode only deletes your <i>local</i> search and browsing history - just the content on your computer. Websites, search engines, internet service
+    Incognito mode only deletes your <i>local</i> search and browsing history — just the content on your computer. Websites, search engines, internet service
     providers, and governments can still easily track you across the web.
 </p>
 <p style="margin-bottom: 1em">
-    That's why it's important to use privacy alternatives that don't share your personal information - such as <a href="https://www.duckduckgo.com/">DuckDuckGo</a> for search.
+    That's why it's important to use privacy alternatives that don't share your personal information, such as <a href="https://www.duckduckgo.com/">DuckDuckGo</a> for search.
 </p>
 
 <p style="margin-bottom: 1em">
     Using Incognito mode to keep you private online is kind of like using a bucket to put out a raging fire:<br /><br />
     <center><img src="https://duckduckgo.com/assets/email/fire.jpg" alt="Picture of man putting out fire with small bucket of water" width="200" height="200" style="padding-bottom:15px;" /></center>
 </p>
 <p style="margin-bottom: 1em">
-   If you didn’t know that private browsing isn’t private — you’re not alone! <a href="https://spreadprivacy.com/is-private-browsing-really-private/">67% of people</a> who use private browsing <i>over-estimate</i> the protection that private browsing modes offer.
+   If you didn’t know that private browsing isn’t private, you’re not alone! <a href="https://spreadprivacy.com/is-private-browsing-really-private/">67% of people</a> who use private browsing <i>over-estimate</i> the protection that private browsing modes offer.
```

Comment:
',' needs a space I think.

Reply:
There is a space, so it's `private, you're`

Reply:
So there is - looks like diff presentation is messed up.

:+1:

Reply:
We'll need to look at this again - whatever sort of space this is, it's not recognised by Gecko :)